syntax = "proto3";

import "network_structs.proto";

message PAuthMessage {

    string token = 1;
    string name = 2;
    string email = 3;
    string avatar_url = 4;
}

message PAuthSucceedMessage {

    int64 id = 1;
}


message PErrorMessage {

    int32 code = 1;
    string message = 2;
}

// Contacts:

message PContactsSearchRequest {

    int32 request_id = 1;
    string query = 2;
}

message PContactsSearchResponse {

    int32 request_id = 1;
    repeated PUser users = 2;
}